### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by nam-co** [amiranagram/livewire-sortablejs#7] [Q&A] [answered]
Hi @amiranagram thanks for the work on this package, I was wondering if your planning on sporting it? also if you have tried it with the new release of livewire

Also is there a way to use it with for example: pull: 'clone', put: false, sort: false?

Thanks for your time

**Comment by amiranagram** [comment, marked as answer]
Hi @nam-co, I am planning to support all the features listed in the examples here: https://sortablejs.github.io/Sortable/

I am just too preoccupied with my day job and cannot seem to find time to do a proper release. As soon as I have some free time I'll ship a bunch of features. I have already coded some. Thanks for understanding.

**Reply by amiranagram** [reply]
For example I have already coded the sortable group feature which you can see here: https://livewire-sortablejs.amirrami.com/examples/shared-list, but I haven't released it yet.

**Comment by nam-co** [comment]
Looking good, thanks @amiranagram , I see in your code you already have mention 'clone', so I will wait for nested groups (I think some else already mention), once again thanks for the work, livewire official sortable is not that good, once you're ready I suggest submitting to Laravel news

**Comment by nam-co** [comment]
Hi @amiranagram , can I email you to me@amirrami.com, I got a little collaboration / proposition / exchange maybe your interested

**Reply by amiranagram** [reply]
For sure.
